tests/pytorch/targets.jsonnet

```diff
@@ -15,9 +15,11 @@
 local nightly = import 'nightly/targets.jsonnet';
 local r1_10 = import 'r1.10/targets.jsonnet';
 local r1_11 = import 'r1.11/targets.jsonnet';
+local r1_11 = import 'r1.12/targets.jsonnet';
```

Comment:
r1_12

Reply:
Thanks lol, definitely not used to editing in vim